### DISCUSSION TITLE
Repressors in the heatmap dotplot

### DISCUSSION THREAD

**Discussion by LukasOussoren** [aertslab/scenicplus#443] [Q&A]
I have a question regarding the heatmap dotplot:
![image](https://github.com/user-attachments/assets/da971f1b-9de1-4c68-a274-a1c496f20119)

This figure is from the new tutorial supporting the new scenicplus object etc. It is clear that any regulon with that is either "+/+", "-/+" or "+/-" is classified as an activator in this plot and "-/-" is classified as a repressor. 

How does this make sense, when, in the old tutorial, the regulons were defined as such:

![image](https://github.com/user-attachments/assets/7b47d700-9cd0-40fa-baa2-5c6cd1974c82)

Furthermore, looking at the same type of plot from and older tutorial, all "+_+" regulons were called activators and all "-_+" regulons were called repressors.

Am I missing something or is the plotting function not working as intended?

**Comment by SeppeDeWinter** [comment]
Hi @LukasOussoren 

No you are not missing something, this is a bug! Thanks for reporting.

This should be fixed now: https://github.com/aertslab/scenicplus/commit/3285c867320aa5732f75d7adea846c992595871a.

Can you check?

All the best,

Seppe
